### PR TITLE
Fix FAQ docx ui upload parsing support and reformat docx.py

### PR DESCRIPTION
**Pull request by w5688414**
### PR types

+ Bug fixes 
### PR changes

+ APIs

### Description

+ Support FAQ ui upload parsing support
+ Fix and reformat docx.py for parsing docx files

### CODE REVIEW

File: pipelines/pipelines/nodes/preprocessor/preprocessor.py [hunk 1]
```diff
@@ -316,25 +327,45 @@ def split(
...
-                    text_splits.append(txt)
-
+            # FAQ text process don't need split text into fix lengths
+            if (not split_answers):
+                if split_overlap:
+                    segments = windowed(elements,
+                                        n=split_length,
+                                        step=split_length - split_overlap)
+                else:
+                    segments = windowed(elements,
```
Comment (wawltor) [inline pipelines/pipelines/nodes/preprocessor/preprocessor.py]: 这个if else 看起来是不必要，直接通过split_overlap数值来决定就行

File: pipelines/pipelines/nodes/preprocessor/preprocessor.py [hunk 2]
```diff
@@ -316,25 +327,45 @@ def split(
...
         # create new document dicts for each text split
         documents = []
         for i, txt in enumerate(text_splits):
             doc = deepcopy(document)
             doc["content"] = txt
+
             if "meta" not in doc.keys() or doc["meta"] is None:
                 doc["meta"] = {}
+            if (split_answers):
+                text_arr = doc["content"].split('\t')
```
Comment (wawltor) [inline pipelines/pipelines/nodes/preprocessor/preprocessor.py:350]: split_answers 具体是代表啥含义了？

Comment (w5688414) [reply pipelines/pipelines/nodes/preprocessor/preprocessor.py:350]: FAQ智能问答需要读取文件的问答对，split_answers表示把问答对按照\t分开

File: pipelines/pipelines/nodes/preprocessor/preprocessor.py (hunk 1 above)
Comment (w5688414) [reply pipelines/pipelines/nodes/preprocessor/preprocessor.py]: 已修改

**Review by wawltor — APPROVED** [review]
LGTM

**Pull request merged**